lib/ExpensiMark.js

```diff
@@ -148,8 +148,13 @@ export default class ExpensiMark {
                 // We also want to capture a blank line before or after the quote so that we do not add extra spaces.
                 // block quotes naturally appear on their own line. Blockquotes should not appear in code fences or
                 // inline code blocks. A single prepending space should be stripped if it exists
-                regex: /\n?^&gt; ?(?![^<]*(?:<\/pre>|<\/code>))([^\v\n\r]+)\n?/gm,
-                replacement: '<blockquote>$1</blockquote>'
+                process: (textToProcess) => {
+                    const regex = new RegExp(
+                        '((\n?^&gt; ?(?![^<]*(?:</pre>|</code>))([^\v\n\r]+))+)',
```

Comment:
You can probably ignore the line length warnings since they're not exactly lines you modified. However this error is interesting. Can you briefly explain to me how you changed the Regex here?

I'm not terribly familiar with this kind of error and am curious why it's calling out these control characters.

Reply:
I tried to reach in a regex to solve the problem without really need use a process function, that was the closest Regex I got, but anyway it doesnt deal with some \n problems. 
At the moment this regex is only used to know if we need to deal with quotes.
I found some people talking about the ESlint problem [here](https://stackoverflow.com/questions/49743842/javascript-unexpected-control-characters-in-regular-expression).
But anyway, changing:
```js
const regex = new RegExp(
                       '((\n?^&gt; ?(?![^<]*(?:</pre>|</code>))([^\v\n\r]+))+)',
                       'gm'
                    );
```
to:
```js
 const regex = new RegExp(
                        /\n?^&gt; ?(?![^<]*(?:<\/pre>|<\/code>))([^\v\n\r]+)\n?/gm
                    );
```
solved the ESlint problem.